packages/std/src/math/uint128.rs

```diff
@@ -208,6 +208,22 @@ impl<'a> ops::Add<&'a Uint128> for Uint128 {
     }
 }
 
+impl ops::Sub<Uint128> for Uint128 {
+    type Output = Self;
+
+    fn sub(self, rhs: Self) -> Self {
+        Uint128(self.u128().checked_sub(rhs.u128()).unwrap())
+    }
+}
+
+impl<'a> ops::Sub<&'a Uint128> for Uint128 {
+    type Output = Self;
+
+    fn sub(self, rhs: &'a Uint128) -> Self {
+        Uint128(self.u128().checked_sub(rhs.u128()).unwrap())
+    }
+}
```

Comment:
Let's keep this out of 0.14.x. The implementation was removed between 0.13 and 0.14 because we wanted to change the return type. We should have at least one version where the functionality does not exist at all such that contracts break the hard way instead of having behaviour changes.

Reply:
Fair. I'll push that to a separate branch. Rebased this one.